### PR TITLE
Update various help commands

### PR DESCRIPTION
**Pull request by panpawn**
This is kind of a standard throughout PS commands - the help commands don't skip a new line from the command that they're the help command to.

### CODE REVIEW

**Comment by TheImmortal** [comment]
>Explains what the + % @ # & next to people's names mean.

So it doesn't explain ~ ?

**Comment by Zarel** [comment]
You might as well fix it for `stathelp` while you're at it.

**Comment by panpawn** [comment]
>You might as well fix it for stathelp while you're at it.

Er, what? Where is that, exactly? :|

**Comment by Zarel** [comment]
Three commands up.

**Comment by panpawn** [comment]
ohh, ``statcalchelp``

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1040,8 +1039,7 @@ exports.commands = {
 			"~ <b>Global Administrator</b> -  They can do anything, like change what this message says"
 		);
 	},
-
-	groupshelp: ["/groups - Explains what the + % @ # & next to people's names mean.",
+	groupshelp: ["/groups - Explains what the symbols before people's names mean.",
```
Comment (Zarel) [inline chat-plugins/info.js]: I'd prefer `the symbols (like % and @) before`

**Comment by panpawn** [comment]
Updated

**Pull request merged**